src/bundle/Resources/views/locationview.html.twig

```diff
@@ -0,0 +1,14 @@
+<h1>{{ location.contentInfo.name }} (Location #{{ location.id }})</h1>
+
+<p>
+    <a href="{#{ path('proto_draft_create', {contentId: location.contentInfo.id}) }#}">Edit this content</a>
```

Comment:
Indeed. While at it, I have added the basic code that iterates over the fields and displays them :)